### ISSUE TITLE
Enable advanced SMB options if BGSource is xDrip and is smoothing data

### ISSUE THREAD

**Issue by jotomo** [MilosKozak/AndroidAPS#911]
- [ ] Retrieve info from xDrip about applied filters
- [ ] Disable SMB when noise is high
- [ ] Disable SMB if noise is negative (n/a due to missed readings or when calibrating)
...

**Comment by winni67** [comment]
Yes, that is really necessary. The native Dexcom algorithm doesn‘t work for me. Please introduce this feature.
